src/IK/include/BipedalLocomotion/IK/QPInverseKinematics.h

```diff
@@ -179,7 +179,9 @@ class QPInverseKinematics : public IntegrationBasedIK
      * |:---------:|:------------------------------:|:---------------:|:----------------------------------------------------------------------------------------------:|:---------:|
      * |`TASK_NAME`|             `type`             |     `string`    |   String representing the type of the task. The string should match the name of the C++ class. |    Yes    |
      * |`TASK_NAME`|           `priority`           |       `int`     | Priority associated to the task.  (Check QPInverseKinematics::addTask for further information) |    Yes    |
-     * |`TASK_NAME`|            `weight`            | `vector<double>`|        Weight associated to the task. It is required only if the task is low priority          |     No    |
+     * |`TASK_NAME`|     `weight_provider_type`     |     `string`    |  String representing the type of the weight provider. The string should match the name of the C++ class. It is required only if the task is low priority          |     No    |
```

Comment:
From the code, it seems that it is not strictly required. Is it worth mentioning what happens if you don't specify it?